Automattic/pocket-casts-ios
# Fixes opening of shared link for episodes with the current position

Fixes #5 

Open podcasts episodes at the correct position when user shares a link with the current position.

## To test

- Start the app 
- Get a shared link for Pocket Casts that share the episode and the current position. Example:`https://pca.st/s1h4dvtr?t=881.0`
- Open this link in Safari on the device/simulator
- Check that the app opens the podcast/episode and moves the current position to the time provided

One thing to keep in mind is that if you are currently listening to that episode the system will not move your current time.

## Checklist
